lib/carthage_cache/project.rb

```diff
@@ -4,13 +4,15 @@ class Project
 
     attr_reader :cartfile
     attr_reader :project_path
+    attr_reader :archive_dir_name
```

Comment:
What about renaming this to `archive_base_path` to be consistent with my suggestion in the config object?